Example/PageBuilderCountdown/view/base/web/js/content-type/example-countdown/countdown.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/**
+ * Sets the countdown fields with remaining time
+ */
+function setCountdownField(target, countdownField, intervalId) {
```

Comment:
A JSDoc comment here would help illustrate what to pass into each parameter

Reply:
done